src/System.Collections.NonGeneric/tests/CollectionBaseTests.cs

```diff
@@ -242,8 +242,10 @@ public static void CopyTo_Invalid()
         {
             MyCollection collBase = CreateCollection(100);
             var fooArr = new Foo[100];
-            Assert.Throws<ArgumentOutOfRangeException>("dstIndex", () => collBase.CopyTo(fooArr, -1)); // Index < 0
-            Assert.Throws<ArgumentException>(string.Empty, () => collBase.CopyTo(fooArr, 50)); // Index + fooArray.Length > collBase.Count
+            // Index < 0
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("destinationIndex", "dstIndex", () => collBase.CopyTo(fooArr, -1));
+            // Index + fooArray.Length > collBase.Count
+            AssertExtensions.Throws<ArgumentException>("destinationArray", string.Empty, () => collBase.CopyTo(fooArr, 50));
```

Comment:
Nit: above where an empty string was needed, ```""``` was used, and here ```string.Empty``` is used.  Let's just try to be consistent.